src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objectOfStyle = {};
```

Comment:
The variable name 'objectOfStyle' could be improved for better readability. Consider using a name that reflects its purpose, such as 'styleObject' or 'cssProperties'.